views/list.hbs

```diff
@@ -0,0 +1,6 @@
+<ul>
```

Comment:
This looks good for now, I will begin working on the list view rendering the roles from the database